src/parser.js

```diff
@@ -73,7 +73,13 @@ const parser = yargs
   })
   .coerce('prettier', config => {
     if (typeof config === 'object') {
-      return camelcaseKeys(config)
+      const cfg = camelcaseKeys(config)
+      return Object.keys(cfg).reduce(
+        key =>
+          cfg[key] =
+            cfg[key] === 'true' || (cfg[key] === 'false' ? false : cfg[key]),
+        {},
+      )
```

Comment:
I agree with @Gandem, I'd prefer to use `boolify` if it's really as simple as: `boolify(camelcaseKeys(config))` which appears to be the case.